Sources/SwiftKuery/Table.swift

```diff
@@ -68,11 +69,15 @@ open class Table: Buildable {
                 if column.isPrimaryKey {
                     columnsWithPrimaryKeyProperty += 1
                 }
-            }
-            else if let label = child.label, label == "tableName" {
+            } else if let label = child.label, label == "tableName", name == nil {
```

Comment:
Having sat and looked at this how do you feel about updating to be:
```
else if let label = child.label, label == "tableName" {
                _name = child.value as! String
            }
        }
        
        defer {
            verifyTableProperties()
        }
        
        guard let name = name else {
            return
        }
        _name = name
```

Reply:
I'm ambivalent about this, but if you're comfortable with it, then I'll make the change :)

Reply:
Done.